packages/components/dropdowns/dropdown-menu/src/dropdown-menu.tsx

```diff
@@ -40,6 +47,48 @@ export type TDropdownMenuProps = {
   children: ReactNode;
 };
 
+function getScrollableParent(element: HTMLElement | null): HTMLElement | null {
```

Comment:
We want to block the scroll of the nearest ancestor with scrolling enabled, if any.